macros/tests/json.rs

```diff
@@ -15,9 +15,39 @@ mod json_like {
     }
 }
 
+fn json_like_testcase(input: &str, expected: &str) {
+    let tokens = input
+        .parse::<gll::grammer::proc_macro::TokenStream>()
+        .unwrap();
+
+    let result = json_like::Value::parse(tokens);
+    let result = match &result {
+        Ok(result) => format!("{:#?}", result),
+        Err(gll::grammer::parser::ParseError { at, expected }) => {
+            format!("{:?}: error: expected {:?}", at, expected)
+        }
+    };
+
+    // HACK(eddyb) clean up the result, as we have no span info.
```

Comment:
We should just bite the bullet and run tests with `--cfg procmacro2_semver_exempt`.

Reply:
What I don't get is why don't they always have the `fmt::Debug` implementation enabled.

Reply:
Also this doesn't work on stable Rust FWIW.